data/templates/default.html5

```diff
@@ -27,8 +27,7 @@ $for(header-includes)$
   $header-includes$
 $endfor$
 $if(math)$
-$if(mathjax)$
-  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+$if(mathjax)$https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
```

Comment:
This doesn't look right.  Isn't the `<script` part missing?  And a newline?  Shouldn't it be like the html4 template?

Reply:
I'll just make the commit directly because it's clear what is needed.